examples/bevy_ggrs/src/box_game.rs

```diff
@@ -83,20 +88,19 @@ pub fn read_local_inputs(
     commands.insert_resource(LocalInputs::<BoxConfig>(local_inputs));
 }
 
-pub fn setup_scene_system(
+pub fn setup_system(
```

Comment:
```suggestion
pub fn setup_scene(
```
we have more than one setup system in the matchbox, so I'd rather keep scene in here